src/osd/ECBackend.cc

```diff
@@ -2356,6 +2356,10 @@ int ECBackend::be_deep_scrub(
 
   utime_t sleeptime;
   sleeptime.set_from_double(cct->_conf->osd_debug_deep_scrub_sleep);
+  if (sleeptime != utime_t()) {
+    lgeneric_derr(cct) << __func__ << " sleeping for " << sleeptime << dendl;
+    sleeptime.sleep();
```

Comment:
this isn't a new problem, but it makes me notice that these sleeps need fixing - since scrub is being called from the op queue threads, sleeping like this blocks one of them.

They should be converted to async sleeps like in PG::scrub().

Reply:
nevermind, this is just a debug sleep